### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by glasnt**
Work in progress, PR-ing for review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,38 +6,90 @@ REST interface to vaultaire
...
 
 The format for output is [time, value] where:
+
 * time: is the unix timestamp, with extra precision in the form of a real number
-* value: is the value represented
+* value: is the value represented. Integer, or float if `_float` is flagged.
+
+Sample API call: 
+
+		sieste.local/raw/ABC123/XYZ456AA?start=1400000000&end=141000000`
```
Comment (olorin) [inline README.md]: This might read a bit better with just the endpoint and no hostname.

File: README.md [hunk 2]
```diff
@@ -6,38 +6,90 @@ REST interface to vaultaire
...
 
 ### /raw/{origin}/{address}
+
 Renders points straight from [readerd](https://github.com/anchor/vaultaire/blob/master/lib/Vaultaire/Reader.hs) via [broker](https://github.com/anchor/vaultaire/blob/master/lib/Vaultaire/Broker.hs), no interpolation is done.
-* `address`: the unique identifier of a source (within an origin)
-* `start`: unix epoch  for start of search as an integer, no value before this will be returned.
-* `end`: unix epoch for end of search as an integer, no value after this will be returned.
+
+
+
```
Comment (olorin) [inline README.md:73]: Whitespace creep here.

**Comment by olorin** [comment]
Are you sure the relevant param is _float? I thought it was as_double.

File: README.md [hunk 3]
```diff
@@ -6,38 +6,86 @@ REST interface to vaultaire
...
+	    [[1400000000,10],[1400000010,20],[1400000030,30]...]
+
+Sample result if using `_float`: 
+
+	    [[1400000000,10.1],[1400000010,20.2],[1400000030,30.3]...]
+
 
 ### /raw/{origin}/{address}
+
 Renders points straight from [readerd](https://github.com/anchor/vaultaire/blob/master/lib/Vaultaire/Reader.hs) via [broker](https://github.com/anchor/vaultaire/blob/master/lib/Vaultaire/Broker.hs), no interpolation is done.
```
Comment (olorin) [inline README.md]: This bit should be updated, readerd isn't a thing anymore.

**Comment by glasnt** [comment]
It is `as_double`. Whoops. Will go through and work out if anything else is out of sync.

**Comment by olorin** [comment]
I think this could also use a bit more semantic information - like that the default number of results is 64, and that queries are 'match' rather than 'search' (i.e., query `bar` will not match value `foobar`, but query `*bar` will), and that queries of the form `*aleph*beth*` would be parsed into two separate wildcard queries such that they could match `*aleph*` and `*beth*` separately in any field, not just together in a single field.

**Comment by glasnt** [comment]
> query `bar` will not match value `foobar`, but query `*bar` will

Updated

> and that queries of the form _aleph_beth\* would be parsed into two separate wildcard queries such that they could match _aleph_ and _beth_ separately in any field, not just together in a single field.

This doesn't currently happen. Searching for `?q=*aleph*beth*` only returns `key~alephbeth`, not `key~aleph,key~beth`. However, `?q=*aleph*&q=*beth*` will return both

If this is not the expected behaviour, then this is a bug.

**Comment by glasnt** [comment]
Close old PR. No longer have commit bit to merge.

**Pull request closed without merging**